MurungaKibaara/Stackoverflow-lite
# Ft user login 162571593

*What does this PR do?*

Adds a UI that allows registered users to sign in.

*Description of task to be completed?*

• Write HTML for the user signin page
• Write CSS to style the signin page

*How should you manually test this?*

1. Clone this repository
2. Cd into the StackOverflow-lite folder
3. Open signin.html using any browser (preferably google chrome)

*What are the relevant pivotal tracker stories?*

#162571593 A user should be able to log in.

*screenshot*
![screenshot from 2018-12-12 18-37-59](https://user-images.githubusercontent.com/43511880/49881048-73db9d00-fe3e-11e8-981a-1d79d9fa3acd.png)
